docs/hello_nextflow/03_hello_gatk.md

```diff
@@ -177,7 +169,7 @@ nextflow run hello-gatk.nf
 Should produce something like:
 
 ```console title="Output"
-N E X T F L O W  ~  version 23.10.1
+N E X T F L O W  ~  version 24.02.0-edge
```

Comment:
```suggestion
N E X T F L O W  ~  version 24.04.4
```
Let's not use an old edge version

Reply:
I don't know how to set the version in the gitpod

Reply:
If it's the version you have in gitpod, fine by me.

Otherwise:
```
export NXF_VER=24.04.4; nextflow self-update
```